server/tso/tso.go

```diff
@@ -278,7 +281,9 @@ func (t *timestampOracle) getTS(leadership *election.Leadership, count uint32) (
 		}
 
 		resp.Physical = current.physical.UnixNano() / int64(time.Millisecond)
+		t.mux.Lock()
```

Comment:
don't need this lock

Reply:
I have tried to remove this lock but the race detecting will warn about this.

Reply:
em, how does race detector report the issue?